controllers/nodes.go

```diff
@@ -232,6 +244,33 @@ func (n *Nodes) deamonsetForMondoo(m *v1alpha1.MondooAuditConfig, cmName string)
 }
 
 func (n *Nodes) Reconcile(ctx context.Context, clt client.Client, scheme *runtime.Scheme, req ctrl.Request, inventory string) (ctrl.Result, error) {
+
+	log := ctrllog.FromContext(ctx)
+	var image string
+
+	if n.Mondoo.Spec.Nodes.Image.Name == "" || n.Mondoo.Spec.Nodes.Image.Tag == "" {
+		image = "docker.io/mondoolabs/mondoo:latest"
+	} else {
+		image = n.Mondoo.Spec.Nodes.Image.Name + ":" + n.Mondoo.Spec.Nodes.Image.Tag
+	}
```

Comment:
I think we want to allow mixing and matching image and/or tag overrides. This allows someone to say "I'm okay with the default mondoo container image, but I just want a specific version/tag", and "I have a custom mondoo image that I need to test with, so I want to specify everything".

So something like this:
```suggestion
	mondooImage := "docker.io/mondoolabs/mondoo"
	mondooTag := "latest"
	if n.Mondoo.Spec.Nodes.Image.Name != "" {
		mondooImage = n.Mondoo.Spec.Nodes.Image.Name
	}
	if n.Mondoo.Spec.Nodes.Image.tag != "" {
		mondooTag = n.Mondoo.Spec.Nodes.Image.Tag
	}
	mondooContainer := mondooImage + ":" + mondooTag
```

Reply:
Sure. I just don't know anyone who would specify a tag but not an image or visa versa. I'll make the change.

Reply:
Allowing users to configure either image or tag is great. so that users can just pin the version but do not need to set the image name!